### PR TITLE
Updated HTTP client library section with further details and instructions

### PR DESCRIPTION
**Pull request by williamexperian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -601,16 +626,50 @@ WebHttpRequest.builder()
         .build()
 ```
 
-### Send http request through WebHttpClient
+### Send the HTTP request through WebHttpClient
+``` java
+client.sendAsync(request);
+```
+
+Calling the sendAsync method returns a WebHttpResponse object.
```
Comment (yudhistira-arya) [inline README.md]: Returns a `CompletableFuture<WebHttpResponse>`

File: README.md [hunk 2]
```diff
@@ -601,16 +626,50 @@ WebHttpRequest.builder()
...
+- `getStatus()`
+- `getMessage()` 
+- `getBody()`
+- `getHeaders()`
+
+An example: 
+
+``` java
+WebHttpResponse response = client.sendAsync(request); 
+String responseBody = response.getBody();
```
Comment (yudhistira-arya) [inline README.md]: Is this one correct? I thought send async will return `CompletableFuture<WebHttpResponse>`?

Comment (yudhistira-arya) [reply README.md]: Something like:

```java
        client.sendAsync(request)
                .thenApply(response -> {
                    final String responseBody = response.getBody();
                    // do processing;
                })
                .exceptionally(e -> {
                   // error handling 
                 });
```

File: README.md [hunk 3]
```diff
@@ -562,7 +580,14 @@ WebHttpClient.builder()
         .build()    
 ```
 
-### Create http GET request object
+### Create an HTTP request
+The requests available are:
```
Comment (izzulhaziq) [inline README.md]: The target endpoint's http method is defined in the request object. Thus, we should say the request supports the following http methods as suggested bellow. Up to you if you want to include the explanation on the method is defined in the request object.

```suggestion
The request supports the following http methods: 
```

File: README.md (hunk 1 above)
Comment (williamexperian) [reply README.md]: Thank you Yudhi, I have updated the README :)

File: README.md (hunk 2 above)
Comment (williamexperian) [reply README.md]: Thank you Yudhi, I have updated the README :)

**Pull request merged**